opensrp-giz-malawi/src/test/java/org/smartregister/giz/util/GizUtilsTest.java

```diff
@@ -149,4 +152,66 @@ public void testCreateChildGrowthEventFromRepeatingGroupShouldInvokeGrowthProces
             }
         }
     }
+
+    @Test
```

Comment:
Update `all` test to match the convention.

Reply:
Done